index.js

```diff
@@ -1,9 +1,10 @@
 module.exports = robot => {
   robot.on('issues.opened', async context => {
-    const options = context.repo({path: '.github/ISSUE_REPLY_TEMPLATE.md'});
-    const res = await context.github.repos.getContent(options);
-    const template = new Buffer(res.data.content, 'base64').toString();
-
-    return context.github.issues.createComment(context.issue({body: template}));
+    const config = await context.config('ISSUE_REPLY_TEMPLATE.yml');
```

Comment:
This will break for [existing users that have a `.github/ISSUE_REPLY_TEMPLATE.md`](https://github.com/search?q=filename%3AISSUE_REPLY_TEMPLATE.md+in%3A.github&ref=opensearch).

Here's my current thinking on config patterns:

- For plugins that predominantly have templated content, I think I prefer using a `.md` file to be consistent with the built-in support for `ISSUE_TEMPLATE.md` and `PULL_REQUEST_TEMPLATE.md`.
- For plugins that have multiple config values, I prefer the a YAML file

WDYT? We should have a more in-depth chat soon about config patterns and document best practices.